plugin.video.francetv/resources/lib/kodilogging.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
```

Comment:
You might want to look into spdx headers (as an update alternative for all these verbose ones). Check the Kodi source files if you're interested. This is not a change request.

Reply:
I'll change my headers for the next release ;)